css/images.css

```diff
@@ -0,0 +1,18 @@
+.image-container {
+  width: 100%;
+}
+
+.image {
+  width: 100%;
+  height: auto;
+}
+
```

Comment:
This is fine. But if you set `max-with: 100%` in `.image`, you would not need an image container.

Reply:
Solved